packages/neo-one-typescript-concatenator/src/__data__/helpers/testDirectoryPath.ts

```diff
@@ -0,0 +1,30 @@
+import { normalizePath } from '@neo-one/utils';
+import * as appRootDir from 'app-root-dir';
+import * as path from 'path';
+
+export const testDirectoryPath = (directory: string) =>
+  normalizePath(
+    path.resolve(
+      appRootDir.get(),
```

Comment:
Ah, I see that you're using it here now - then it should be in `devDependencies`, not `dependencies`.